Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/EdgeConfiguration.cs

```diff
@@ -138,16 +137,6 @@ private void DisableSpecCompliance (ExtendedEdgeOptions edgeOptions)
       edgeOptions.AddAdditionalCapability ("w3c", false);
     }
 
-    /// <summary>
-    /// Starting with Edge version 79, msedgedriver no longer supports the default <see cref="DriverOptions.BrowserName"/> value of
-    /// "chrome" set in <see cref="ChromeOptions"/>, instead "MicrosoftEdge" must be specified. This should be obsolete when RM-7275
-    /// is resolved.
-    /// </summary>
-    private void AdjustBrowserNameCapability (ExtendedEdgeOptions edgeOptions)
-    {
-      s_browserNameSetter.Value.Invoke (edgeOptions, new object[] { "MicrosoftEdge" });

```

Comment:
You can remove `s_browserNameSetter` itself as well on line 56

Reply:
Whoops.